apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -465,6 +465,24 @@ const aichatSlice = createSlice({
       state.fieldVisibilities =
         levelAichatSettings?.visibilities || DEFAULT_VISIBILITIES;
     },
+    resetToDefaultAiCustomizations: (
```

Comment:
I was wondering if we could use existing reducers for this, but I think having it as its own reducer (with a single argument that has everything it needs) makes sense!